docs/SchemeFileFormat.md

```diff
@@ -78,15 +78,15 @@ For example: `my_set/prod_ns/deny_all_policy`. If there are multiple policies na
+|------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|
+| outputFormat     | Output format specification.                                                                                                                                                                                                                                                   | string [ txt / yaml / csv / md / dot / jpg/ txt_no_fw_rules] |
+| outputPath       | A file path to redirect output into.                                                                                                                                                                                                                                           | string                                                       |
+| outputEndpoints  | Choose endpoints type in output.                                                                                                                                                                                                                                               | string [ pods / deployments ]                                |
+| subset           | A dict object with the defined subset elements to display in the output                                                                                                                                                                                                        | [subset](#subset) object                                     |
+| fullExplanation  | Choose if to print all counterexamples causing the query result in the output                                                                                                                                                                                                  | bool                                                         |
+| excludeIPv6Range | If the policies of the config do not contain any IPv6 addresses, do not include IPv6 range in the query results                                                                                                                                                                | bool  [default: True]                                        |
+| explain          | A pair of node names to explain the policies affecting their connection or lack of it. Relevant only for connectivity query. Connections including IP-Blocks will show only the configurations of the node in that connection (since, IP-Blocks does not have configurations). | string [ ns/node1,ns/node2 ]                           |
```

Comment:
```suggestion
| explain          | A pair of node names (comma separated) to explain the policies affecting their connection or lack of it. Relevant only for connectivityMap query. Connections including IP-Blocks will show only the configurations of the node in that connection (since, IP-Blocks does not have configurations). | string [ ns/node1,ns/node2 ]                           |
```
In scheme file the query is named `connectivityMap`

Reply:
Done